pentaho/src/main/java/pt/webdetails/cgg/CggService.java

```diff
@@ -81,7 +82,9 @@ public Response draw(
     @DefaultValue( "0" ) @QueryParam( "height" ) Long height,
     @Context HttpServletResponse servletResponse, @Context HttpServletRequest servletRequest ) {
 
-    this.draw( script, type, outputType, attachmentName, multiChartOverflow, null,
+    String normalizedScriptPath = script == null ? null : Paths.get( script ).normalize().toString().replace( "\\", "/" );
+
```

Comment:
Why do the normalization here and not in the other `#draw` method overload, together with other path normalization which is already being performed there, such as slash normalization?

Reply:
Ah 🤦‍♂️  I had it there before and then removed it, and then put it back in the wrong place, that is the correct solution